src/node_kernels/SDRSSTAMSNodeKernel.C

```diff
@@ -90,10 +99,36 @@ SDRSSTAMSNodeKernel::execute(
   const NodeKernelTraits::DblType om_fOneBlend = 1.0 - fOneBlend;
   const NodeKernelTraits::DblType beta =
     fOneBlend * betaOne_ + om_fOneBlend * betaTwo_;
-  const NodeKernelTraits::DblType gamma =
-    fOneBlend * gammaOne_ + om_fOneBlend * gammaTwo_;
   const NodeKernelTraits::DblType sigmaD = 2.0 * om_fOneBlend * sigmaWTwo_;
 
+  NodeKernelTraits::DblType gammaOne_apply;
+  NodeKernelTraits::DblType gammaTwo_apply;
+  // apply limiter to gamma
+  if (lengthScaleLimiter_) {
+    // calculate mixing length
+    const NodeKernelTraits::DblType l_t = stk::math::sqrt(tke)/(stk::math::pow(betaStar_, .25)*sdr);
+ 
+    // calculate maximum mixing length
+    const NodeKernelTraits::DblType l_e = .00027*referenceVelocity_/corfac_;
```

Comment:
Could you add a comment on where `0.00027` comes from?

Reply:
Yes.